chatterbot/chatterbot.py

```diff
@@ -14,21 +14,22 @@ def __init__(self, name, **kwargs):
         self.name = name
         kwargs['name'] = name
 
-        storage_adapter = kwargs.get('storage_adapter',
-            'chatterbot.adapters.storage.JsonFileStorageAdapter'
-        )
+        default_storage = {"storage_class": 'chatterbot.adapters.storage.JsonFileStorageAdapter'}
+        storage_kwargs = kwargs.get('storage_adapter', default_storage)
+        storage_adapter = storage_kwargs.get('storage_class')
+        database = storage_kwargs.get('database')
 
-        logic_adapters = kwargs.get('logic_adapters', [
-            'chatterbot.adapters.logic.ClosestMatchAdapter'
-        ])
+        default_logic = [{"logic_class": 'chatterbot.adapters.logic.ClosestMatchAdapter'}]
+        logic_kwargs = kwargs.get('logic_adapters', default_logic)
+        logic_adapters = [dic['logic_class'] for dic in logic_kwargs]
 
-        input_adapter = kwargs.get('input_adapter',
-            'chatterbot.adapters.input.VariableInputTypeAdapter'
-        )
+        default_input = {"input_class": 'chatterbot.adapters.input.VariableInputTypeAdapter'}
+        input_kwargs = kwargs.get('input_adapter', default_input)
+        input_adapter = input_kwargs.get('input_class')
 
-        output_adapter = kwargs.get('output_adapter',
-            'chatterbot.adapters.output.OutputFormatAdapter'
-        )
+        default_output = {"output_class": 'chatterbot.adapters.output.OutputFormatAdapter'}
+        output_kwargs = kwargs.get('output_adapter', default_output)
+        output_adapter = output_kwargs.get('output_class')
```

Comment:
What are your thoughts on changing `output_class`, `input_class`, `storage_class` and `logic_class` all just to `adapter_class` to keep the parameters consistent across all adapters?